mtp_api/apps/credit/managers.py

```diff
@@ -6,6 +8,9 @@
 from credit.constants import LOG_ACTIONS, CREDIT_STATUS, CREDIT_RESOLUTION
 
 
+logger = logging.getLogger(__name__)
```

Comment:
The issue with having every file log under the same namespace is that it can be difficult to trace back to where the logging statement was emitted. Is there a reason for having all files use the same logging namespace?

Reply:
the logging settings explicitly set how the 'mtp' logger should work and there are special test functions which reference it by name. and this file doesn't use it anway.

Reply:
Fair enough. I did want to bring up test assertions against log statements, and maybe talk about what value they bring to the tests. I feel like asserting against log statements and other ancillary developer-facing functionality leads you to testing functions rather than functionality; what do you think the chances of that test catching an unintentional removal of a log statement are vs catching an intentional removal/modification of that same log statement?